frontend/src/Components/NewRunConfig/NewRunState.tsx

```diff
@@ -0,0 +1,38 @@
+import { showToast } from "../Toaster";
+import AdvancedSettingsMenu from "./AdvancedSettings/AdvancedSettings";
+import Settings from "./Settings/Settings";
+import InitConfigState from "../../Helpers/SimConfig";
+
+function request(configJSON: string) {
+  const requestOptions = {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: configJSON,
+  };
+  var host = window.location.protocol + "//" + window.location.host;
+  fetch(`${host}/simulate`, requestOptions)
+    .then(function (response) {
+      return response.json();
+    })
+    .catch(function (error) {
+      console.log("There has been a problem with your fetch operation: " + error.message);
+    });
```

Comment:
The return handler doesn't really fit since you're returning a promise but not handling it anywhere. I suggest removing it completely
The catch can pop a toaster with "danger" intent to display the error message to the user